compiler/rustc_middle/src/hir/map/collector.rs

```diff
@@ -62,65 +56,48 @@ fn hash_body(
     stable_hasher.finish()
 }
 
-/// Represents an entry and its parent `HirId`.
-#[derive(Copy, Clone, Debug)]
-pub struct Entry<'hir> {
-    parent: HirId,
-    node: Node<'hir>,
+pub(super) fn collect(
+    sess: &'a Session,
+    krate: &'hir Crate<'hir>,
+    definitions: &'a definitions::Definitions,
+    mut hcx: StableHashingContext<'a>,
+    owner: LocalDefId,
+) -> Option<IndexedHir<'hir>> {
+    let item = *krate.owners.get(owner)?.as_ref()?;
+    let hash = hash_body(&mut hcx, item);
+    let mut nodes = IndexVec::new();
```

Comment:
```suggestion
    let mut nodes = IndexVec::with_capacity(1);
```

Similar thing here
https://github.com/rust-lang/rust/blob/e7f7fe462a54b1caeb804a974cd43ba9fd7bee5c/compiler/rustc_middle/src/hir/map/collector.rs#L108-L109

But i don't think this will give observable results, really.